### ISSUE TITLE
The Tag component X button should have screen reader only content "Remove" instead of "Close"

### ISSUE THREAD

**Issue by nicklemmon** [SparkPost/matchbox#502]
## 🚀 Feature request

**Current Behavior**

Currently, the `<Tag/>` component with a dismiss button renders with the screen reader only content "Close" - which sounds a little confusing when describing the typical interaction with a dismissable `<Tag/>`.

**Desired Behavior**

Update the content to more accurately describe the interaction.

**Suggested Solution**

Update the content from "Close" to "Remove" (or something else!)

**Who does this impact? Who is this for?**

Screen reader users.
